### ISSUE TITLE
Inconsistency between clients upload method

### ISSUE THREAD

**Issue by andreapinti** [qusly/qusly-core#43]
Hi, and thanks for your work!
I was looking at the difference between the upload method in Client class and ConcurrentClient class:
```
Client.upload(path, source, [, options])
```
```
ConcurrentClient.upload(localPath, remotePath)
```
Arguments are inverted and it could be annoying if you want to switch between clients in a programmatic way.

**Comment by xnerhu** [comment]
Hello! Thank you for your appreciation! This is a very good point.

**Comment by xnerhu** [comment]
Released in `v2.5.2`.
